note-qualifying-protocols-and-encodings.md

```diff
@@ -0,0 +1,47 @@
+# Qualifying Protocols and Encodings
+
+The explicit goal of the CloudEvents effort, as expressed in the specification, is 
+"describing event data in a common way" and "to define interoperability of event 
+systems that allow services to produce or consume events, where the producer and 
+consumer can be developed and deployed independently".
```

Comment:
I believe for the group to achieve its goals it may want to look at a single encoding and transport protocol for all producers. Say JSON and HTTP 1.1 / 2 (Although true push for eventing may provide some challenges). This way would provide a least common denominator and consistency across cloud providers and ecosystems. This is a MUST to honor the spec. 

For specialized consumers that prefer different encodings or transports, they could be referenced here based on status and use in the ecosystem. 

Any provider could produce events in different encodings and transports, but to be compliant they must produce via HTTP and JSON (Or whatever the chosen encoding and transport shall be).

Reply:
@derekcollison could you open an issue for your idea? I'd like to track it separately from this PR because I think it applies regardless of whether or not we define a "bar" for new encodings/transports (this PR).  Do I have that right?

Reply:
Sure no problem.

Reply:
Done #288

Reply:
thanks!